ECHO-COVID19.ipynb

```diff
@@ -844,8 +704,7 @@
    },
    "outputs": [],
```

Comment:
The map has some locations that are almost definitely wrong (e.g. north pole). Not sure if there's anything to be done there.

Also the % of unmappable facilities is astoundingly large! Might be worth making a pie chart of just that

Reply:
Yea, I'm certainly not going to try to fix EPA/industry's own geocoding errors.

I've noticed the unmappable number and want to triple check it to be sure.